log/log.go

```diff
@@ -57,12 +57,18 @@ func buildConfig(c zap.Config) (*zap.Logger, error) {
 func configureLogger(l *zap.Logger, service string) (Logger, error) {
 	l = l.With(zap.String("service", service))
 
-	rollbarClean := rollbar.Setup(l.Sugar().With("pkg", "log"), service)
 	cleanup := func() {
-		rollbarClean()
 		_ = l.Sync()
 	}
 
+	if _, ok := os.LookupEnv("ROLLBAR_TOKEN"); ok {
```

Comment:
LookupEnv will return true when `ROLLBAR_TOKEN=`, better to use os.Getenv or env.Get() and compare against `""`

Reply:
I feel like if we do any validation of the contents we should do as much validation as we possibly can (length, format, etc) -- but I don't think we should do that, so I did what you suggest  ;)